coordinator-common/src/test/java/org/apache/kafka/coordinator/common/runtime/EventAccumulatorTest.java

```diff
@@ -83,15 +83,15 @@ public void testBasicOperations() {
         assertNull(accumulator.poll());
 
         List<MockEvent> events = Arrays.asList(
-            new MockEvent(1, 0),
-            new MockEvent(1, 1),
-            new MockEvent(1, 2),
-            new MockEvent(2, 0),
-            new MockEvent(2, 1),
-            new MockEvent(2, 3),
-            new MockEvent(3, 0),
-            new MockEvent(3, 1),
-            new MockEvent(3, 2)
+                new MockEvent(1, 0),
```

Comment:
Why did you change this? We use 4 spaces everywhere in this module.

Reply:
My bad, IntelliJ must have reformatted that, undoing